### PR TITLE
Add a note about when metrics was added

### PR DESCRIPTION
**Pull request by jaredly**

### CODE REVIEW

**Review by mholt — CHANGES_REQUESTED** [review]
To clarify, metrics have existed for a while in Caddy; they only became opt-in with that global option in 2.6.

**Comment by jaredly** [comment]
ohh interesting

File: src/docs/markdown/metrics.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ In other words, you're going to want to be able to _monitor_ Caddy.
 
 You'll need to turn metrics on. If using a Caddyfile, enable metrics [in global options](/docs/caddyfile/options#metrics):
 
+> **Note**
+> Prior to Caddy 2.6, metrics were on by default, so this global configutaion is unnecessary. In 2.6+, they are opt-in for performance reasons.
```
Comment (mholt) [inline src/docs/markdown/metrics.md]:
```suggestion
> Caddy 2.6 made metrics opt-in. To enable them, use either [Caddyfile global options](https://caddyserver.com/docs/caddyfile/options#metrics) or [the `metrics` parameter](https://caddyserver.com/docs/json/apps/http/servers/metrics/) of a server's JSON config.
```

**Review by mholt — CHANGES_REQUESTED** [review]
Close -- it's more like this, I'd say!

**Review by mholt — APPROVED** [review]
Thanks for the PR!

**Pull request merged**